### PR TITLE
Apply French language contributions

### PR DESCRIPTION
**Pull request by habemuscode**
**Related to**

https://oat-sa.atlassian.net/browse/BSA-97

**Description**

Change translations here:

https://github.com/oat-sa/extension-tao-community/blob/4c1c7f3597ea9a0778483cdf3391481c9cbf83fd/locales/fr-FR/messages.po#L123

Screenshot_18.png
image

### CODE REVIEW

**Comment by jsconan** [comment]
Please assign PR only when they are ready for a review.
This is producing noise on the todo list.

File: locales/fr-FR/messages.po [hunk 1]
```diff
@@ -120,7 +120,7 @@ msgid "Item(s)"
 msgstr "Item(s)"
 
 msgid "Items"
-msgstr "Items"
+msgstr "Articles"
```
Comment (wazelin) [inline locales/fr-FR/messages.po:123]: `messages_po.js` has the same identifier.
I'm pretty sure it should be updated as well.

Comment (jsconan) [reply locales/fr-FR/messages.po:123]: Yes, you must run `php tao/scripts/taoTranslate.php -a=compile -e=taoCe -l=fr-FR` in order to get the `message_po.js` file updated.

More info here: https://hub.taotesting.com/articles/documentation-for-core-components/internationalization-i18n#how-to-update-or-add-translations

Comment (habemuscode) [reply locales/fr-FR/messages.po:123]: message_po.js generated and I update for all languages

**Pull request merged**